projects/components/src/common/error/error-banner.component.scss

```diff
@@ -4,6 +4,6 @@
  */
 
 clr-alert ::ng-deep .alert {
-    margin-left: 24px;
-    margin-right: 24px;
+    margin-left: 0;
```

Comment:
Can you rebase off the most recent origin/master? This shouldn't show up in this diff because you did it in another commit.

Reply:
Thanks. I rebased off master to take care of this.